Gemfile

```diff
@@ -77,7 +77,7 @@ gem "jbuilder", require: false
 gem "web-console", require: false
 
 # Action Pack and railties
-rack_version = ENV.fetch("RACK", "~> 2.0") # Change to ~> 3 after #46594 is merged.
```

Comment:
Is the CI passing 2.x on this env? if not this will make so 2.0 isn't tested anymore. It doesn't seems it is https://github.com/rails/buildkite-config/blob/main/pipeline-generate#L308